lint-manifest.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# vim: set ft=sh
+
+set -e
+
+#
+# Validate environment
+#
+if [[ ! -d "pipeline-config" || ! -d "lint-manifest" ]]; then
```

Comment:
You don't need this, concourse will complain if the inputs specified in the task definition don't exist